test/regression/run1/data/parameters.ini

```diff
@@ -2,7 +2,6 @@
 shb_id=15
 num_threads=1
 tau=1
-model=original
```

Comment:
Okay, so this is actually a big problem. If we remove this field, then there is no way of telling which regression tests correspond to which model structure, and the only remaining place where that information exists is buried in the `RunRegressionTests.sh` script. I think we may have to switch back to taking the model structure from the parameters file, and only use the command line options for the data pipeline and the location of local data files.

Reply:
We could do both? The way the code works is to read the files for options first, then overwrite them if command line arguments are given.

Reply:
Okay, that sounds fine.

Reply:
I have done this by moving both enums up by 1, i.e. option `0` is now a default that is unlabelled and can be used to determine if the option has been set: 
```
// Only set model structure to default of "original" if no option specified
// in parameters file
if(input_params.model_structure == ModelStructureId(0))
{
    input_params.model_structure = ModelStructureId::ORIGINAL;
}

// Only overwrite this model structure option if command line argument is not empty
if(_args.structure != ModelStructureId(0))
{
    input_params.model_structure = _args.structure;
}
```